.vscode/settings.json

```diff
@@ -90,7 +99,9 @@
 	"[markdown]": {
 		"editor.defaultFormatter": "esbenp.prettier-vscode"
 	},
+	"editor.wordWrapColumn": 120,
+	"editor.wordWrap": "wordWrapColumn",
```

Comment:
personally don't mind this and am not affected since i don't use vscode :) 
does this mean other folks who use vscode will start refactoring files, or is this just affecting new lines added?

Reply:
This is just going to affect the display in vscode:

Before:
![Screenshot 2023-04-25 at 8 26 07 AM](https://user-images.githubusercontent.com/58678/234308448-5a53be10-ab39-4be8-829b-1adeb597aa48.png)


After:
![Screenshot 2023-04-25 at 8 25 49 AM](https://user-images.githubusercontent.com/58678/234308471-4eca94a4-6c16-4a17-b208-74ec8010fc6a.png)

Notice the return values wrap at 120 columns.


Ideally, `gofmt` would handle wrapping the line for us but it looks like that's contentious (see https://github.com/golang/go/issues/11915). 

Googling around, I found [golines](https://github.com/segmentio/golines) ([blog post](https://yolken.net/blog/cleaner-go-code-golines)) which would extend `gofmt` to wrap the lines at a certain line length as well as formatting key/value pairs. 

I think there's a lot of merit to adding golines since we definitely have a lot of functions that have far too many params (that we should probably refactor into a struct) but may be difficult to change at the moment. Obviously adding that in now would be a lot of changes so adding it in as a vscode change is a nice middle ground in the short term.